validationFunctions/validatePartners.test.js

```diff
@@ -15,65 +15,118 @@ describe("Function: validatePartners", () => {
   it("Should return false if array length is above 5", () => {
     //Arrange
     const badPartnersLength = [
-      { name: "", is_primary_contact: true },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false }
+      "jon",
+      "Mike",
+      "jo",
+      "dan",
+      "wes",
+      "exa",
     ];
     //Act
-    const valid = validatePartners(badPartnersLength);
+    const valid = validatePartners(
+      badPartnersLength
+    );
     //Assert
     expect(valid).toBe(false);
   });
 
   it("Should return false if array length is below 2", () => {
     //Arrange
-    const badPartnersLength = [{ name: "", is_primary_contact: true }];
+    const badPartnersLength = ["was"];
     //Act
-    const valid = validatePartners(badPartnersLength);
+    const valid = validatePartners(
+      badPartnersLength
+    );
     //Assert
     expect(valid).toBe(false);
   });
 
   it("Should return true when the array length is between 2-5", () => {
     //Arrange
     const goodPartnersLength = [
-      { name: "", is_primary_contact: true },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false }
+      "jon",
+      "Mike",
+      "jo",
+      "dan",
     ];
     //Act
-    const valid = validatePartners(goodPartnersLength);
+    const valid = validatePartners(
+      goodPartnersLength
+    );
     //Assert
     expect(valid).toBe(true);
   });
 
   it("Should return true when the array length is exactly 2", () => {
     //Arrange
-    const goodPartnersLength = [
-      { name: "", is_primary_contact: true },
-      { name: "", is_primary_contact: false }
-    ];
+    const goodPartnersLength = ["jon", "exa"];
     //Act
-    const valid = validatePartners(goodPartnersLength);
+    const valid = validatePartners(
+      goodPartnersLength
+    );
     //Assert
     expect(valid).toBe(true);
   });
 
   it("Should return true when the array length is exactly 5", () => {
     //Arrange
     const goodPartnersLength = [
-      { name: "", is_primary_contact: true },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false },
-      { name: "", is_primary_contact: false }
+      "jon",
+      "Mike",
+      "jo",
+      "dan",
+      "exa",
+    ];
+    //Act
+    const valid = validatePartners(
+      goodPartnersLength
+    );
+    //Assert
+    expect(valid).toBe(true);
+  });
+});
+
+describe("Function: validatePartners", () => {
```

Comment:
No need for separate `describe()` block here, just add these tests the the one above.